### ISSUE TITLE
Multiple preloads

### ISSUE THREAD

**Issue by grahamperrin** [simonfrey/faster-pageload-web-extensions#4]
With this enabled: 

☑ Preload every visible link

– I get multiple preloads. 

At <https://instant.page/tech> for example, after loading the page then pointing at the *Intensity* and *Pages not preloaded* links: 

![image](https://user-images.githubusercontent.com/192271/71448304-46544f80-2732-11ea-946e-11ce617afe2a.png)

Maybe a consequence of this? 

* Mozilla bug [1527334 - Navigating to a page that was prefetched with <link rel="prefetch"> re-downloads it](https://bugzilla.mozilla.org/show_bug.cgi?id=1527334)

– mentioned at https://www.reddit.com/r/firefox/comments/e5v4wq/came_across_this_extension_to_make_the_page_load/fa92xib/?context=2

**Comment by simonfrey** [comment]
Yes this seems to be a problem with FF itself. Will keep an eye on the issue
